src/main/scala/monocle/POpticConstructors.scala

```diff
@@ -0,0 +1,70 @@
+package monocle
+
+import monocle.impl._
+import monocle.functions.*
+import monocle.internal.*
+
+
+trait POpticConstructors:
+  def modify[S, T, A, B](_modify: (A => B) => S => T): POptic[Modify, S, T, A, B] = 
+    POptic(
```

Comment:
should we define these implementation in the companion object of the Impl class? e.g.

```scala
def modify[S, T, A, B](_modify: (A => B) => S => T): POptic[Modify, S, T, A, B] = 
  POptic(SetterImpl(_modify))

object SetterImpl {
  def apply[S, T, A, B](_modify: (A => B) => S => T): SetterImpl = ...
}
```